app/src/main/java/io/homeassistant/companion/android/sensors/BluetoothSensorManager.kt

```diff
@@ -136,10 +135,10 @@ class BluetoothSensorManager : SensorManager {
         if (checkPermission(context, bluetoothConnection.id)) {
 
             val bluetoothDevices = BluetoothUtils.getBluetoothDevices(context, true)
-            pairedDevices = bluetoothDevices.filter { b -> b.paired }.map { it.address }
-            connectedPairedDevices = bluetoothDevices.filter { b -> b.paired && b.connected }.map { it.address }
-            connectedNotPairedDevices = bluetoothDevices.filter { b -> !b.paired && b.connected }.map { it.address }
-            totalConnectedDevices = bluetoothDevices.filter { b -> b.connected }.count()
+            pairedDevices = bluetoothDevices.filter { b -> b.paired }.map { "${it.address} (${it.name})" }
```

Comment:
If we don't have a name, the `name` property will be set to the device's address. This could result in a string like "A1:B2:C3:D4:E5:F6 (A1:B2:C3:D4:E5:F6)". 

This seems undesirable to me, so perhaps add a check it is not the same (or make name nullable, if possible)?

Reply:
ok I will opt to only add it only if we have a name, this method is used in other places so I would rather not modify it :)